tasks/scanner-cli/v3/task.json

```diff
@@ -1,8 +1,8 @@
 {
     "id": "9F57024B-31F9-4E58-9E39-A47CCC098F03",
     "name": "SonarQubeScannerCli",
-    "friendlyName": "SonarQube  Scanner CLI",
-    "description": "Perform a SonarQube analysis of the source code of your PHP, JavaScript... projects",
+    "friendlyName": "Scanner CLI",
+    "description": "Perform a SonarQube or SonarCloud analysis of the source code of your PHP, JavaScript... projects",
     "helpMarkDown": "[More Information](http://redirect.sonarsource.com/doc/install-configure-scanner-tfs-ts.html)",
```

Comment:
Maybe `Version: 3.0.2.` shoule be added here for every v3 tasks, it was added by the build script before.

Reply:
I don't like duplicating the version manually. In case we need to update tasks in the future, this is likely we will forgot. What about doing it automatically with gulp?

Reply:
Ok, so what do you think ?
For v3 taking the current version field of the task and use that to update `helpMarkDown` ?
Or add a new parameter like `--v3Version 3.0.15` for exemple ?

Reply:
I would take current version of the task if possible.